src/main/java/bowling/domain/BowlingGame.java

```diff
@@ -0,0 +1,30 @@
+package bowling.domain;
+
+import bowling.domain.frame.Frames;
+import bowling.domain.player.Player;
+
+public class BowlingGame {
+
+    private Frames frames;
+
+    public BowlingGame(String playerName) {
+        Player player = new Player(playerName);
+        this.frames = new Frames(player);
+    }
+
+    public void addPoint(int point) {
+        if (frames.gameOver()) {
```

Comment:
`BowlingGame`의 메서드를 사용해도 되겠네요. 

```suggestion
        if (gameOver()) {
```